src/Emulator/Peripherals/Peripherals.csproj

```diff
@@ -1,4 +1,3 @@
-<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I assume those changes were introduced automatically by the IDE.

Could you keep only the line that adds the source file, but move it to line 396?

Reply:
This is a duplicated comment, closing.